02-analytics/technical.md

```diff
@@ -0,0 +1,377 @@
+| Event ID      | event              | Hardcoded: `share`                  |
+| Page Name     | pageName           | same ID as for page/screen tracking |
+| Element Type  | elementType        | episode, series, ...                |
+| Element ID    | elementId          | series, episode ID                  |
+
+## Other Tracking events
+
+This tracking events do not have data besides the common fields. The list contains
+the `EventID` values to be used
+
+| Event              | Event ID          |
+|--------------------|-------------------|
+| Log In             | login             |
+| Log Out            | logout            |
+| Airplay started    | airplayStarted    |
+| ChromeCast started | chromecastStarted |
+
+## Video events
+
+The video events should, in addition to the common data, also contain the following.
+
+### Common video data table
+
+The names are specified by Rudderstack and are inconsistent with the rest of the
+document. They should be used as they are here (camelCase).
+
+| Data            | Name               | Comments                                                                                                     |
+|-----------------|--------------------|--------------------------------------------------------------------------------------------------------------|
+| Session ID      | sessionId         | regenerate after 30 minutes of not being used                                                                |
+| Is Live         | livestream         | true/false                                                                                                   |
+| Episode ID      | contentPodId     |                                                                                                              |
+| Positing        | position           | From start of video in seconds                                                                               |
+| Total Length    | totalLength       | null if live stream                                                                                          |
+| Video Player    | videoPlayer       | Name of the player (VideoJS, ...)                                                                            |
+| Is Full screen? | fullScreen        |                                                                                                              |
+| Quality         | quality            | if available                                                                                                 |
+| Volume          | volume             | in %                                                                                                         |
+| Has video       | hasVideo          | false if in "audio only mode"                                                                                |
+| Subs            | subtitlesLanguage | null if not possible to determine. "OFF" if turned off. "N/A" if subs are not available in selected language |
```

Comment:
"N/A" will not happen in mobile app, because either subtitles are enabled in some language or disabled "OFF"